whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+Much of LSST cosmology may be limited by systematic rather than statistical
+errors.  This is especially true of weak gravitational lensing, which relies on
+very accurate (\ie low bias), but very noisy, measurements of the shapes of many
+galaxies, and high signal-to-noise measurements of PSF calibration stars.  As
+outlined in the SRD, uniformity of seeing in the bands used for weak lensing and
+special observing strategies are required in order to reduce additive and
+multiplicative shear systematics.
 
 Achieving the ultimate sensitivity of the LSST to weak lensing science places
-stringent requirements on our ability to accurately measure galaxy shapes and redshifts,
-which in turn demands precise and accurate knowledge of the point spread function,
-astrometry, and photometry. These measurements are influenced by the interaction of
-light with the Earth’s atmosphere, the telescope optics, and the CCD sensors. Sysematics
-in the shear are introduced in each case.   Methods have been developed for suppressing 
-these systematics in current lensing surveys. These and new methods will be applied to 
-the LSST survey.
-
-Over the sample of 3-4 billion galaxies, the shear systematics must be below
-one part in 10,000 for additive shear, and one part in 1000 for multiplicative shear.
-Each visit to a sky patch encounters these systematics. Some observing strategies can 
-effectively randomize these over all visits to a field.  Below we discuss the observing 
-strategies for suppressing shear systematics and metrics for their success.
+stringent requirements on our ability to accurately measure galaxy shapes and
+redshifts, which in turn demands precise and accurate knowledge of the point
+spread function, astrometry, and photometry.  These measurements are influenced
+by the interaction of light with the Earth's atmosphere, the telescope optics,
+and the CCD sensors.  Systematics in the shear are introduced in each case.
+Observing strategies have been developed for suppressing these systematics in
+current lensing surveys, such as the Deep Lens
+Survey\footnote{\url{dls.physics.ucdavis.edu}}.  These and new methods will be
+applied to the LSST survey.
 
-\subsection{Target Selection}
+To leading order, we can express the effect of shear systematics in the observed
+shear $\gamma^\mathrm{obs}_i$ as a small linear perturbation of the true shear
+$\gamma_i$,
 
-Image quality must be uniformly good in the bands used for weak lens shear.  These will be 
-the $r$ and $i$ bands.   Depending on the current weather and seeing, the scheduler
-will have a list of priorities for next-field, based on prior history of coverage.
-Nearby fields in need of coverage in these bands will be given high priority if the
-seeing is better than some specified value, likely 0.7 arcsec FWHM.
+$$ \gamma_i^\mathrm{obs} = (1+m_i) \gamma_i + c_i, $$
 
+where $m_i$ is the multiplicative and $c_i$ is the additive systematic in the
```

Comment:
Indeed. I prefer to stick with what is measured in the galaxy shear, and 
then point out that the systematic source is mainly the PSF!

On 6/29/16 6:46 PM, Rachel Mandelbaum wrote:
> In whitepaper/Cosmology/wl.tex
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478#discussion_r69058070>:
>
>>
>> +where $m_i$ is the multiplicative and $c_i$ is the additive systematic in the
>
> I think it's more informative to use |a_i \gamma_PSF| or something that
> clearly indicates that to lowest order, for most methods, the additive
> contamination is a linear function of the PSF ellipticity. This makes it
> more clear that we prefer observing strategies that cause the average
> PSF ellipticity at a given point (over all exposures) to average towards
> zero.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LSSTScienceCollaborations/ObservingStrategy/pull/478/files/68564bd57311f22658256026e5e6e6a39c2554f4#r69058070>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACA6YqrE4VsbKsUXuDq864DcXB8U0Gm4ks5qQx_rgaJpZM4I9hd->.
>